cohortextractor/backends/__init__.py

```diff
@@ -1,7 +1,8 @@
 # All backends need to be imported here so they get registered
 from .base import BACKENDS
+from .databricks import DatabricksBackend
 from .graphnet import GraphnetBackend
 from .tpp import TPPBackend
 
 
-__all__ = ("BACKENDS", "TPPBackend", "GraphnetBackend")
+__all__ = ("BACKENDS", "DatabricksBackend", "TPPBackend", "GraphnetBackend")
```

Comment:
We need this otherwise for the backend registration in `BaseBackend.__init_subclass__` to work